lib/synced/model.rb

```diff
@@ -73,6 +73,7 @@ def synced(strategy: :updated_since, **options)
       self.synced_auto_paginate         = options.fetch(:auto_paginate, true)
       self.synced_transaction_per_page  = options.fetch(:transaction_per_page, false)
       self.synced_handle_processed_objects_proc  = options.fetch(:handle_processed_objects_proc, nil)
+      self.synced_endpoint              = options.fetch(:endpoint, self.table_name)
```

Comment:
this is a bit different than before - we expect it to have `table_name` method implemented, while before we only tableized class name. Not sure if anybody would use it in a different way but why change it?

Reply:
so we can define the endpoint as a symbol instead of a string. looks more consistent and makes sense as we make a symbol out of it anyway.